carbon/components/OptionsSwitcher/index.tsx

```diff
@@ -19,6 +19,7 @@ export default function OptionsSwitcher<T extends Key>(props: {
     return props.onSelectionChange(value);
   };
   return (
+    // FIXME: this doesn't let you override certain styles
     <ul
       className={`${styles.list} ${!!props.className ? props.className : ""}`}
```

Comment:
@Atmosfearful I couldn't get the override working with emotions cx here. My guess is that this is because we don't use emotion to define the styles. I had to resort to `!important` to get the style override working.